manta-js/package/src/utils.ts

```diff
@@ -63,4 +63,16 @@ export class MantaUtilities {
       console.error(e);
     }
   }
+
+  /// Removes `numBytes` many leading bytes from a `hexString` 
+  /// `useQuotes` specifies whether quotation marks should be added
+  static removeLeadingBytesFromHexString(hexString: string, numBytes: number, useQuotes:boolean): string {
```

Comment:
If it's just a method used in the example, can it be moved to the example?

Reply:
My idea is to have it more easily available for the users. Otherwise they'd have to copy the function.
I'll move it to the example if you prefer though.

Reply:
Make sense, If the Dapp may use this method, then add it to the SDK

Reply:
The dApp is not using it yet. But other dApps that want to use the SDK with cross-chain transfers will need to.